lib/active_admin/async_exporter/reports/dsl.rb

```diff
@@ -14,10 +14,16 @@ def csv_async(decorate_model: false)
           csv_report(columns: csv_fields, decorate_model: decorate_model)
         end
 
-        def csv_column(column_name, column_value = nil)
+        def csv_column(column_name, column_value = nil, &block)
           column_value ||= column_name
 
-          csv_fields[:"#{column_name.to_sym}"] = column_value.to_sym
+          if block_given?
+            # For CR: Raise error if block.params.blank?
```

Comment:
With params you mean the `x` here `{ |x| ... }` ?

Reply:
Exactly, since I parsed the string based on the first pipe of the variable sent to the proc, I need to always have at least one param, othwerwise I wouldn't be able to parse the code. However, if we use a regexp like you said we might be able to accept any params

Reply:
Hmm okay. I think we should throw an error...

Reply:
Done!